playbooks/roles/ocp-coo/vars/main.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 
 coo_e2e_env:
-  PATH: "{{ ansible_env.PATH }}:/usr/local/go/bin"
+  PATH: "{{ ansible_env.PATH }}:/usr/local/chainsaw"
```

Comment:
Are we not using the variable `chainsaw_path`?

Reply:
ahh yes, I have made the changes, can you please review the changes.
Thanks.